### ISSUE TITLE
Rename ebt_base_parameters

### ISSUE THREAD

**Issue by richfitz** [traitecoevo/plant#136]
Needs to be `FFW16_ebt_base_parameters` now I think.  Will affect all projects

**Comment by richfitz** [comment]
Could do:

```r
ebt_base_parameters <- function(model) {
  switch(model,
    FFW16=FFW16_ebt_base_parameters(),
    stop("Unknown model ", model))
}
```

which has the nice property of not breaking any existing code.

**Comment by richfitz** [comment]
Closed in favour of #133
